nullaway/src/main/java/com/uber/nullaway/dataflow/AccessPath.java

```diff
@@ -255,6 +256,38 @@ public static AccessPath getAccessPathForNodeWithMapGet(Node node, @Nullable Typ
     }
   }
 
+  /**
+   * Gets corresponding AccessPath for a class field element.
+   *
+   * @param field The class field element.
+   * @param receivers The list of receivers.
+   * @return corresponding AccessPath.
+   */
+  public static AccessPath fromFieldAndBase(Element field, List<Element> receivers) {
+    List<AccessPathElement> elements = new ArrayList<>();
+    elements.add(new AccessPathElement(field));
+    Root root;
+    Element rootReceiver;
+    if (receivers == null || receivers.size() == 0) {
+      root = new Root();
+    } else {
+      rootReceiver = receivers.get(0);
+      assert !(rootReceiver instanceof MemberSelectTree);
+      if (rootReceiver == null
+          || rootReceiver.toString().equals("super")
+          || rootReceiver.toString().equals("this")) {
+        root = new Root();
+      } else {
+        root = new Root(rootReceiver);
+      }
+      receivers.remove(0);
```

Comment:
Don't mutate the list (it's an unexpected and rare thing to do).  Just change the next loop to go from index 1 to the end